src/model/user/user.ts

```diff
@@ -0,0 +1,16 @@
+export interface IUser
+{
+    _id: string;
+    firstName: string;
+    lastName: string;
+    email: string;
+    hashedPassword: string;
+    bio: string;
+    recipes: { [keyof: string]: Date }
+
+createdAt: Date;
+updatedAt: Date;

```

Comment:
I think the formatting here was missed by the linter?

Reply:
I just committed the changes straight from github